helm_deploy/apply-for-legal-aid/requirements.lock

```diff
@@ -1,7 +1,6 @@
 dependencies:
 - name: postgresql
-  repository: https://kubernetes-charts.storage.googleapis.com/
-  condition: postgresql.enabled
-  version: 0.15.0
-digest: sha256:baab52c09113374d85d22a45f6606c4f86cb20cfbb77be7a8f2f065fc08c0ff9
-generated: 2018-08-21T13:06:22.312036363+01:00
+  repository: https://charts.bitnami.com
+  version: 6.2.0
+digest: sha256:c1d53c0e210361522bcd20e01c0c6a5ad680858145d328476b457daa3a1711dd
+generated: 2019-07-26T13:58:57.870599881+01:00
```

Comment:
The original PostgreSQL image runs as root so we are now using bitnami's PostgreSQL
as recommended by cloud-platform.
It's also what is used by `check-financial-eligibility` at the moment